api/v1alpha1/openapi.yaml

```diff
@@ -1087,6 +1087,163 @@ paths:
         "404":
           description: NotFound
           content: {}
+  /api/v1/templateversions:
```

Comment:
this isn't part of /api/vi/fleet? could you add details of the API in the description and use case please :pray:

Reply:
I put an explanation in the commit message, but added it here as well.